src/libcore/default.rs

```diff
@@ -84,10 +84,33 @@
 
 use marker::Sized;
 
-/// A trait for giving a type a useful default value.
+/// A trait for giving a type a useful default value. For more information, see
+/// [the module-level documentation][module].
```

Comment:
Same here with regards to summaries, I'd move this second sentence into its own paragraph.

Or, ideally, move that module-level documentation into the type itself. It's always been a bit tricky/scattered to tell where things should go; I'm slowly becoming of the opinion that this kind of thing is an antipattern.